nixos/modules/system/boot/luksroot.nix

```diff
@@ -594,6 +625,26 @@ in
             '';
           };
 
+          tryEmptyPassphrase = mkOption {
+            default = false;
+            type = types.bool;
+            description = ''
+              If keyFile fails then try an empty passphrase first before
+              prompting for password.
+            '';
+          };
+
+          keyFileTimeout = mkOption {
+            default = null;
+            example = 5;
+            type = types.nullOr types.int;
+            description = ''
+              The amount of time in seconds for a keyFile to appear before
+              timing out and trying passwords. This is required if tryEmptyPassphrase
+              is enabled and systemd in initrd is enabled.
```

Comment:
I don't think this description is correct. This is not required if `tryEmptyPassphrase` is enabled.